site/content/blog/2020-07-27-automated-ui-testing-with-cypress.md

```diff
@@ -0,0 +1,147 @@
+
+Start by adding metadata, as we call it, in a test file.
+```javascript
+// Stage: @prod
+// Group: @accessibility
+```
+Then, simply initiating `node run_tests.js --stage='@prod' --group='@accessibility` will run production tests for accessibility groups.
+
+For sure, there are lots of best practices out there which are not mentioned here but I hope it helps anyone reading this, especially for those who want to get started with Cypress or set up an Automated UI Testing in general.
+
+## Conclusion
+The setbacks and hurdles we experienced are not necessarily limitations in the Cypress test framework, but manageable through organization and best practices. We are glad to have it as part of our automated UI testing and very much thankful to Cypress for creating a tool that helps make writing E2E enjoyable.
+
+## Thanks to all contributors!
+Finally, I’d like to thank all the contributors who shaped and helped set up our E2E and put us where we are today.
+
+__All E2E contributors and their number of contributions (as of this writing):__
+Abdulrahman (Abdu) Assabri (5), Abraham Arias (6), Adzim Zul Fahmi (3), Agniva De Sarker (1), Alejandro García Montoro (5), Allen Lai (3), Andre Vasconcelos (1), Anindita Basu (2), Arjun Lather (3), Asaad Mahmood (2), Ben Schumacher (2), bnoggle (1), Bob Lubecker (32), Brad Angelcyk (1), Brad Coughlin (3), catalintomai (2), cdncat (1), Christopher Poile (1), Clare So (2), Claudio Costa (3), Clément Collin (1), composednitin (1), Cooper Trowbridge (2), Courtney Pattison (2), d28park (2), Daniel Espino García (24), David Janda (1), Devin Binnie (5), Donald Feury (1), Eli Yukelzon (5), Farhan Munshi (7), Guillermo Vayá (5), Harrison Healey (14), Hossein Ahmadian-Yazdi (10), Hyeseong Kim (1), Jesse Hallam (6), Jesús Espino (2), Jonathan Rigsby (2), Jorde G (1), Joseph Baylon (68), Kelvin Tan YB (1), kosgrz (2), lawikip (2), m3phistopheles (1), Marc Argent (3), Maria A Nunez (1), Mario de Frutos Dieguez (3), Martin Kraft (11), Matthew Shirley (1), Md Zubair Ahmed (11), Md_ZubairAhmed (1), metanerd (1), Miguel Alatzar (1), NiroshaV (1), oliverJurgen (2), Patrick Kang (1), Pradeep Murugesan (1), Prapti (4), Rob Stringer (1), Rohitesh Gupta (49), Romain Maneschi (3), Sam Wolfs (1), Sapna Sivakumar (3), Saturnino Abril (160), Scott Bishel (5), Shota Gvinepadze (1), sij507 (7), Soo Hwan Kim (2), sourabkumarkeshri (1), Sudheer (7), syuo7 (1), Takatoshi Iwasa (1), Tomas Hnat (1), Tsilavina Razafinirina (3), Valentijn Nieman (3), VishalSwarnkar (3), Vladimir Lebedev (11), VolatianaYuliana (2), Walmyr (1), 興怡 (1)
+
+## Ready to get started with Cypress?
+If Cypress sounds interesting to you, please join us for <a target="_blank" href="https://forum.mattermost.org/t/cypress-test-automation-hackfest-kickoff/10191">Cypress Test Automation Hackfest</a> in August and win an exclusive Mattermost swag bag!
```

Comment:
@jasonblais @JustinReynolds-MM @lindalumitchell - added note about the hackfest, similar to the one from blog post.
<img width="821" alt="Screen Shot 2020-07-28 at 2 03 38 AM" src="https://user-images.githubusercontent.com/5334504/88575595-b7683680-d076-11ea-8b29-92051311a067.png">

Reply:
I like it! Though the Hackfest is now, in the last week of July, not August. :)

Reply:
Good catch! I've seen the correction too in the blog post - updated to `which runs through August`